rmw_implementation/src/functions.cpp

```diff
@@ -355,6 +355,10 @@ RMW_INTERFACE_FN(rmw_service_server_is_available,
   rmw_ret_t, RMW_RET_ERROR,
   3, ARG_TYPES(const rmw_node_t *, const rmw_client_t *, bool *))
 
+RMW_INTERFACE_FN(rmw_set_log_severity,
+  rmw_ret_t, RMW_RET_ERROR,
+  1, ARG_TYPES(const rmw_log_severity_t *))
```

Comment:
Since `rmw_log_severity_t` is a primitive type this doesn't need to be passed as `const *`. A plain `rmw_log_severity_t` is sufficient.